cdp_scrapers/instances/portland.py

```diff
@@ -116,7 +117,16 @@ def get_person(self, name: str) -> Optional[Person]:
         #       Should add to known_persons so we don't query this same person again.
         return None
 
-    def get_matter(self, event_page: BeautifulSoup) -> Optional[Matter]:
+    def separate_name_from_title(self, title_and_name: str):
+        # title_and_name:
+        #   The title (Mayor of Commissioner) and name of a Portland City Commission
+        #   member.
+        #   e.g., Mayor Ted Wheeler, Commissioner Carmen Rubio
+
+        name_index = title_and_name.find(" ")
+        return title_and_name[name_index + 1 :]
+
+    def get_matter(self, minute_section: BeautifulSoup) -> Optional[Matter]:
```

Comment:
Isn't `minute_section` a `Tag` Shak? Or is `Tag` a `BeautifulSoup` (i.e. inherits from) and we can leave this as-is. I'll check real quick.

Reply:
Yeah I believe the correct type for `minute_section` is `Tag`. So here and... `get_votes()` (I may be missing some other method) we should have `minute_section: Tag`. Do you agree?

Reply:
Yes, you're correct. It's a `Tag`, and I fixed it in other places.